rumqttc/src/lib.rs

```diff
@@ -145,6 +144,8 @@ use rustls_native_certs::load_native_certs;
 pub use state::{MqttState, StateError};
 #[cfg(any(feature = "use-rustls", feature = "use-native-tls"))]
 pub use tls::Error as TlsError;
+use tokens::Resolver;
```

Comment:
```suggestion
pub use tokens::Resolver;
```

Could we make the resolver public for testing?